core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -69,7 +69,6 @@ oppia.directive('audioBar', [
 
           $scope.isAudioBarAvailable = function() {
             return (
-              BrowserCheckerService.supportsAudioPlayback() &&
               $scope.languagesInExploration.length > 0);
```

Comment:
@sshou14, can we move this to line 72? (following 80 chars a line)

Reply:
Thanks for the catch!